dts/arm/microchip/mec172x/mec172xnsz-pinctrl.dtsi

```diff
@@ -750,6 +750,10 @@
 		pinmux = < MCHP_XEC_PINMUX(011, MCHP_AF2) >;
 	};
 
+	pwm4_alt_gpio001: pwm3_alt_gpio001 {
+		pinmux = < MCHP_XEC_PINMUX(001, MCHP_AF1) >;
```

Comment:
mec172xnsz pinctrl dtsi corresponds to 144 pin package, where gpio001 doesn't exist. This can be removed.